src/robot/result/autoexpandkeywordmatcher.py

```diff
@@ -0,0 +1,53 @@
+#  Copyright 2008-2015 Nokia Networks
+#  Copyright 2016-     Robot Framework Foundation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from robot.errors import DataError
+from robot.model import TagPatterns
+from robot.utils import MultiMatcher, is_list_like, py2to3
+
+
+def validate_autoexpandkeywords(options):
+    for opt in options:
+        low = opt.lower()
+        if not (low.startswith('name:') or
+                low.startswith('tag:')):
```

Comment:
`str.startswith()` accepts a tuple of strings to match. This could be shortened to `opt.lower().startswith(('name:', 'tag:'))`. 

Validation is so simple that having it in `settings.py` could be considered as well.

Reply:
changed + checked required python version = 2.5, thus it's safe to use.

Reply:
applied improvement, but i prefer to stay validation in that file beacuse of SOLID design principle